central/role/resources/list.go

```diff
@@ -89,10 +148,23 @@ func newResourceMetadata(name permissions.Resource, scope permissions.ResourceSc
 	return md
 }
 
-func newResourceMetadataWithFeatureFlag(name permissions.Resource, scope permissions.ResourceScope, flag features.FeatureFlag) permissions.ResourceMetadata {
```

Comment:
nit:
The `newResourceMetadataWithFeatureFlag` might be worth keeping in commented form if we're to introduce new resources with feature flag later on.
Alternately, new resources can be added and not wired until the associated features are activated.